jujugui/static/gui/src/app/app.js

```diff
@@ -948,11 +948,11 @@ YUI.add('juju-gui', function(Y) {
       Renders the Log out component or log in link depending on the
       environment the GUI is executing in.
     */
-    _renderLoginOutLink: function() {
+    _renderUserMenu: function() {
       const controllerAPI = this.controllerAPI;
       const linkContainerId = 'profile-link-container';
       const linkContainer = document.getElementById(linkContainerId);
-      if (linkContainer === null) {
+      if (!linkContainer) {
```

Comment:
Why was this change made? `getElementById` returns `null` if no element is found.